CMakeLists.txt

```diff
@@ -758,6 +758,7 @@ endif()
 add_dependencies(buildtests_cxx stress_test)
```

Comment:
Normally when adding a new dependency, you need to add a section to CMakeLists.txt.template
(similar to this: 
https://github.com/grpc/grpc/blob/d059e1c43ced2279ea4594df04f62761d7b95a86/templates/CMakeLists.txt.template#L176)

Given the inlined vector seems to be header-only, you might get away with only adding the include path 
https://github.com/grpc/grpc/blob/d059e1c43ced2279ea4594df04f62761d7b95a86/templates/CMakeLists.txt.template#L515 (right now you are getting cmake compilation failures as cmake doesn't know anything about third_party/absl/container yet.

Reply:
Actually, you should at least add a comment to CMakeLists.txt.template saying that we're using header-only stuff from abseil so we didn't setup proper dependency the way it's done for other third_party deps.

Btw, seems like abseil only supports building with bazel - so currently we cannot use anything else than header-only stuff unless we want to write Makefile and CMakeLists.txt for them (and we really don't want to).